apps/condo/domains/ticket/utils/serverSchema/TicketDocumentGeneration/paidWorks.js

```diff
@@ -0,0 +1,195 @@
+
+    const { stream } = await buildExportFile({
+        templatePath: `./domains/ticket/templates/ticketDocumentGenerationTemplates/paidWorks/${locale}.docx`,
+        replaces: documentData,
+    })
+
+    return {
+        stream,
+        filename: `paid_works_ticket_${ticket.id}_${formatDateToTimezone(undefined, timeZone, 'DD_MM_YYYY')}.docx`,
+        mimetype: DOCX_FILE_META.mimetype,
+        encoding: DOCX_FILE_META.encoding,
+        meta: {
+            listkey: 'TicketDocumentGenerationTask',
+            id,
+        },
+    }
+}
+
+const generateTicketDocumentOfPaidWorks = async ({ task, baseAttrs, context, locale, ticket, organization }) => {
+    const { iec, psrn, organizationAddress } = await getLocaleToPullFinanceInfoClient(locale, organization)
+
+    const { format, timeZone: timeZoneFromUser } = task
+
+    const timeZone = normalizeTimeZone(timeZoneFromUser) || DEFAULT_ORGANIZATION_TIMEZONE
+    const printDate = dayjs().tz(timeZone).locale(locale)
+
+    const contact = ticket.contact ? await getByCondition('Contact', {
+        id: ticket.contact,
+        deletedAt: null,
+    }) : null
+
+    const employee = organization.id && ticket.executor
+        ? await getByCondition('OrganizationEmployee', {
+            organization: { id: organization.id },
+            user: { id: ticket.executor },
+            deletedAt: null,
+        })
+        : null
+
+    const invoices = await Invoice.getAll(context, {
+        ticket: { id: ticket.id },
+        deletedAt: null,
+        status_not: INVOICE_STATUS_CANCELED,
+    }, 'id currencyCode accountNumber rows { isMin toPay name count vatPercent } recipient { bankName bankAccount bic }', {
+        sortBy: ['createdAt_ASC'],
+    })
+
+    const currencyCode = get(invoices, '0.currencyCode') || DEFAULT_INVOICE_CURRENCY_CODE
+    let totalSum = Big(0), totalVAT = Big(0)
+
+    const listOfWorks = invoices.reduce((acc, invoice) => {
+        const rows = Array.isArray(invoice.rows) ? invoice.rows : []
+        acc.push(...rows.map((row, index) => {
+            const price = Big(!row.isMin ? row.toPay : 0)
+            const sum = price.times(get(row, 'count', 1))
+            const vatPercent = Big(get(row, 'vatPercent', 0))
+            totalSum = totalSum.plus(sum)
+            totalVAT = totalVAT.plus(sum.times(vatPercent).div(100))
+
+            return {
+                number: index + 1,
+                name: row.name || '',
+                count: row.count || '',
+                price: !Number.isNaN(price) ? renderMoney(price, currencyCode, locale) : '',
+                vat: !Number.isNaN(vatPercent) ? vatPercent : '',
+                sum: !Number.isNaN(sum) ? renderMoney(sum, currencyCode, locale) : '',
+            }
+        }))
+
+        return acc
+    }, [])
+
+    // NOTE: There are two different types of data in the document template
+    // The first is data inserted into tables
+    // The second is data inserted directly into the document paragraph
```

Comment:
If you say `data inserted in document paragraph` in docs, you should use this terminology consistently thoroughout the code

const documentDataInParagraph 

or in docs 

// Note: this doc needs two variables to be generated:
// 1. Values that will go into the table (worksData)
// 2. Values that will be embeded into the text of the document (textData)

const documentTextData = { ... }